.tav.yml

```diff
@@ -6,6 +6,11 @@ mysql:
   commands:
     - node test/instrumentation/modules/mysql/mysql.js
     - node test/instrumentation/modules/mysql/pool-release-1.js
+mysql2:
+  versions: ^1.0.0
+  commands:
+    - node test/instrumentation/modules/mysql/mysql.js
+    - node test/instrumentation/modules/mysql/pool-release-1.js
```

Comment:
Looks like these should be mysql2, not mysql.

Reply:
Yea, I ran it locally and it worked (but was actually just running mysql...) so I pushed and then realized I have quite a bit left to fix :)

Reply:
No worries. Thanks for the great work you've done so far! I'll try to get this reviewed as soon as I can, once it's been updated. 👍

Reply:
I was hoping to work toward sharing test cases between node-mysql and node-mysql2, but unfortunately I've run into multiple spots where node-mysql2 isn't a perfect drop-in replacement. The two codebases read very similarly, and many parts of the instrumentation work out of the box, but there are subtle differences that end up breaking some cases.

For example, in node-mysql2 `Connection.query(<a Query>)` uses the Query object without invoking `Connection.createQuery`, but in node-mysql the same call passes the Query object to `Connection.createQuery` which immediately returns it.

The connection factory tests are otherwise mostly working by passing over the case I described above, but the pool tests are dead on arrival - no trace recorded in the basic case.

This is going to take some more work, will revisit.

Reply:
Ah, yeah, that sounds tough. I'd be happy to work through this together, if you like. It looks like we're in the same time zone, so we could sync up on a slack somewhere, if we're on any of the same ones. 😉

Reply:
I'm actually not on any Node-related Slacks - maybe invite me to one if you like.